packages/api/src/external/commonwell/admin/recreate-patients-at-hies.ts

```diff
@@ -103,14 +103,14 @@ export async function recreatePatientAtCW(
 
     // create new patient, including linkint to person and network link to other patients
     log(`Creating new patient at CW...`);
-    const cwIds = await registerAndLinkPatientInCW(
+    // WARNING Could interfere with currently running state of another PD
+    const cwIds = await create({
```

Comment:
why changing this?

Reply:
The idea is that "patient-discovery" is a whole flow, and so we want to enter it from the top-level function (not `registerAndLinkPatientInCW`). Edits to the flow may not hit code that enters the flow midway. Help ensure edits to the PD flow affect all places in the app where it's called.

Not "necessary" though...